packages/dashboard/src/components/composed/Layout.tsx

```diff
@@ -1,8 +1,20 @@
+import { useEffect } from "react";
 import { AppShell, createStyles } from "@mantine/core";
+import {
+  showNotification,
+  updateNotification,
+  hideNotification
+} from "@mantine/notifications";
 import { Outlet } from "react-router-dom";
 import { useDash } from "src/hooks";
 import Sidebar from "src/components/composed/Sidebar";
 import Notice from "src/components/composed/Notice";
+import {
+  analyticsNotifications,
+  analyticsNotificationId
+} from "src/utils/notifications";
+
+const ARBITRARY_ANALYTICS_NEXT_ASK_THRESHOLD = 31544444444; // 365 days
```

Comment:
How'd you do this math?

Reply:
Uh I took 4 significant digits and just... went with it. This is updated! See below.